app/util/number/index.test.ts

```diff
@@ -130,6 +130,62 @@ describe('Number utils :: fromTokenMinimalUnit', () => {
     expect(fromTokenMinimalUnit(1e16, 6)).toEqual('10000000000');
     expect(fromTokenMinimalUnit(1e18, 18)).toEqual('1');
   });
+
+  it('should round number by default', () => {
+    expect(fromTokenMinimalUnit(new BN('1000000000000000000'), 18)).toEqual(
+      '1',
+    );
+    expect(fromTokenMinimalUnit(new BN('10000000000000000000'), 18)).toEqual(
+      '10',
+    );
+    expect(fromTokenMinimalUnit(new BN('100000000000000000000'), 18)).toEqual(
+      '100',
+    );
+    expect(fromTokenMinimalUnit(new BN('1000000000000000000000'), 18)).toEqual(
+      '1000',
+    );
+    expect(fromTokenMinimalUnit(new BN('10000000000000000000000'), 18)).toEqual(
+      '10000',
+    );
+
+    // test decimal greater than 30,000
+    expect(fromTokenMinimalUnit(new BN('50000000000000000000000'), 18)).toEqual(
+      '49999.999999999995805696',
+    );
+
+    // test decimal less than 1e-14
+    expect(fromTokenMinimalUnit(hexToBN('576129d2d21d64a5'), 18)).toEqual(
+      '6.296359739485676544',
+    );
+  });
+
+  it('should not round number if isRounding is false', () => {
```

Comment:
```suggestion
  it('does not round number if isRounding is false', () => {
```
Reference: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test